### DISCUSSION TITLE
Welcome to movie-database Discussions!

### DISCUSSION THREAD

**Discussion by pirateIV** [ben64103/movie_app-using-TMDB-API#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.
  * Fork the Repository .
  * Fix bugs.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by coder0103** [comment]
@ben64103 thanks for the invite
